src/components/RadioButton/RadioButton.vue

```diff
@@ -0,0 +1,190 @@
+<template>
```

Comment:
Everything I commented in this component is part of the ATOMS component. You can remove this one since we're going to replace the old ones. The old v1 components don't need the `RadioButton` component